hoot-core/src/main/cpp/hoot/core/scoring/BaseComparator.cpp

```diff
@@ -68,18 +68,18 @@ void BaseComparator::_calculateColor(double v, double max, QRgb& c) const
     }
     if (v < max / 3.0)
     {
-      int r = v / (max / 3.0) * 255.0;
-      c = qRgb(r, 0, 0);
+      double r = v / (max / 3.0) * 255.0;
+      c = qRgb((int)r, 0, 0);
```

Comment:
These (and the ones below) work but my preference would be to do the following instead:
```
int r = std::static_cast<int>(v / (max / 3.0) * 255.0);
c = qRgb(r, 0, 0);
```
so that the second line stays the same but the calculation contains the cast before assignment.